### ISSUE TITLE
[BUG] Crashing

### ISSUE THREAD

**Issue by SuicidalEeper** [CCBlueX/LiquidBounce#4522]
### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

Nightly 

### Operating System

Windows 11

### Minecraft Version

1.21.1

### Describe the bug

Just testing anticheats in this current scenario and then my game hard a hardcore lagspike and crashes. This has also happened at times when playing on servers where I suddenly have a hardcore lagspike (entire screen freezes) then it crashes

### Steps to reproduce

Unsure what the cause is so not sure how to easily reproduce it

### Client Log

```shell
https://paste.ccbluex.net/a145bf637
```

**Comment by SuicidalEeper** [comment]
This is still happening and I should also add that these crashes are not just the game freezing but it freezes my entire computer for up to a minute (can't tab out, screen sometimes goes completely black, inputs aren't working) and then when the freeze/crash is over all things get processed at once

**Comment by SuicidalEeper** [comment]
Bump (idk if that's allowed) but still happening on latest version too idk I just wanna play the game

**Comment by 1zun4** [comment]
No idea

**Comment by SuicidalEeper** [comment]
> No idea

Uh well that's concerning is there nothing I can do? Started happening 4 days ago now maybe always been using nightly

**Comment by SuicidalEeper** [comment]
Well next time I crash I'll post another log here idk if it will be any different but yeah

**Comment by MukjepScarlet** [comment]
How about the memory usage before crashing?

**Comment by Crefax** [comment]
I have the same problem. Because of this problem, I deleted all the files of the CCBlueX folder. This problem did not occur for 1 day. I just downloaded the jello theme and after a short time, the same problem occurred.
The theme may be incompatible with new features or it's just a big coincidence.
By the way, when the game freezes, the computer also freezes, nvidia instant replay is reset and CPU usage increases from 20% to 70%, processor temperature also increases significantly.

Edit: It doesn't work when you choose the default theme again, but it gets fixed when you restart the game.

**Comment by SuicidalEeper** [comment]
> I have the same problem. Because of this problem, I deleted all the files of the CCBlueX folder. This problem did not occur for 1 day. I just downloaded the jello theme and after a short time, the same problem occurred. The theme may be incompatible with new features or it's just a big coincidence. By the way, when the game freezes, the computer also freezes, nvidia instant replay is reset and CPU usage increases from 20% to 70%, processor temperature also increases significantly.

Yeah both my CPU and GPU go up a very high % during the freezes too unsure if it's with themes never used one myself and still getting the issue
